config/config-template-zap-long.yaml

```diff
@@ -15,6 +15,10 @@ config:
   # all the results of all scanners will be stored under that location
   base_results_dir: "./results"
 
+  # In RapiDAST only: should RapiDAST verify certificates
+  # possible values: true [default], false, /path/to/a/PEM/file
+  tls_verify: true
```

Comment:
I would rename this variable to something more specific that clearly indicates the TLS verification is limited to certain functions, not applied across all of RapiDAST. Maybe something like like `tls_verify_for_downloads`

Reply:
agree. Can we have a clearer name?

Reply:
renamed to "tls_verify_for_rapidast_downloads" , to explicitly indicate that it applies only to RapiDAST made by RapiDAST, not the scanners